spec/spec_helper.rb

```diff
@@ -85,10 +85,6 @@
   original_wd  = Dir.pwd
   original_env = ENV.to_hash
 
-  def pending_jruby_shebang_fix
```

Comment:
Oops, doesn't commited spec/support/builders.rb changes, pushed fixed version.
I assume that `executables` is actually ruby scripts here regardless of RUBY_PLATFORM, so why not check `#/usr/bin/env jruby\n` shebang on JRuby.
These test are passing here w/ `ruby_shebang?` changes.